include/nvector/nvector_petsc.h

```diff
@@ -92,7 +92,7 @@ SUNDIALS_EXPORT N_Vector N_VClone_Petsc(N_Vector w);
 SUNDIALS_EXPORT void N_VDestroy_Petsc(N_Vector v);
 SUNDIALS_EXPORT void N_VSpace_Petsc(N_Vector v, sunindextype *lrw, sunindextype *liw);
 SUNDIALS_EXPORT void N_VSetArrayPointer_Petsc(sunrealtype *v_data, N_Vector v);
-SUNDIALS_EXPORT void *N_VGetCommunicator_Petsc(N_Vector v);
+SUNDIALS_EXPORT MPI_Comm N_VGetCommunicator_Petsc(N_Vector v);
```

Comment:
Should this be `SUNComm`?
```suggestion
SUNDIALS_EXPORT SUNComm N_VGetCommunicator_Petsc(N_Vector v);
```

Reply:
No, its an MPI only vector, there is no reason to use a `SUNComm` here (which would just be a typedef to `MPI_Comm` anyways.